cobra/flux_analysis/phenotype_phase_plane.py

```diff
@@ -2,12 +2,20 @@
 
 from __future__ import absolute_import
 
+from warnings import warn
+from collections import defaultdict
+from itertools import product
 from multiprocessing import Pool
+import pandas as pd
 
 from numpy import (
-    abs, arange, dtype, empty, int32, linspace, meshgrid, unravel_index, zeros)
+    nan, abs, arange, dtype, empty, int32, linspace, meshgrid, unravel_index,
```

Comment:
With this many imports I tend to use `import numpy as np` and then just go with `np.nan` etc.